app/settings.py

```diff
@@ -36,6 +36,7 @@ def _get_value(key, default_value=None):
 FTP_USER = os.getenv('FTP_USER')
 FTP_PASS = os.getenv('FTP_PASS')
 FTP_FOLDER = '/'
+FTP_FEEDBACK_FOLDER = '/EDC_QFeedback'
```

Comment:
If we're adding a new hardcoded folder, should there be another card made to add it to sdx-console?  Not vital but something to consider